### ISSUE TITLE
External display - black screen after patch

### ISSUE THREAD

**Issue by mihaeltomic** [Floris497/mac-pixel-clock-patch-V2#214]
MBP 2017 (10.12.6)
Dell S2817Q (4K)
HDMI 2.0 to USB-C Satechi adapter
Used CoreDisplay-patcher.command

I've patched and I have an issue with Dell having a black screen and not detecting (HDMI 2.0) signal.
Any ideas? 

P.S. also how to unpatch if necessary?

**Comment by Floris497** [comment]
Did you try switchresx? Does it still work on 30Hz? Did you reenable SIP?

The readme states how to unpatch `patch.command unpatch`

**Comment by mihaeltomic** [comment]
1. I tried switchresx but the (2k) resolution is blurry for some reason, so I uninstalled it. 
2. After the patch there is information that my monitor is working on the 60Hz and it is recognized by MacOS in System Information, but there's no signal detection on the monitor itself.
3. I tried with enabled, reenabled and disabled SIP - no luck.
4. At the moment it's unpatched and SIP re-enabled since I need a second monitor
